src/codechicken/nei/recipe/GuiRecipe.java

```diff
@@ -25,9 +25,20 @@
 import java.awt.Rectangle;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.ForkJoinPool;
 
 public abstract class GuiRecipe extends GuiContainer implements IGuiContainerOverlay, IGuiClientSide, IGuiHandleMouseWheel, IContainerTooltipHandler
 {
+    protected static final ForkJoinPool forkJoinPool;
+
+    static {
+        int poolSize = Runtime.getRuntime().availableProcessors() * 2 / 3;
```

Comment:
Let's re-use the existing one if we can, rather than making a new one.

Reply:
OK, done